Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerDevolvedRegulation.cshtml

```diff
@@ -0,0 +1,60 @@
+@page "/organisation/{id}/supplier-to-buyer-devolved-regulation"
+@using CO.CDP.OrganisationApp.Constants
+
+@model CO.CDP.OrganisationApp.Pages.Organisation.SupplierToBuyerDevolvedRegulationModel
+@{
+    ViewData["Title"] = "Do devolved regulations apply to your organisation?";
+    var devolvedHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Devolved)).HasInnerHtml;
+
+    var backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-devolved-regulation";
+
+}
+
+<a href="@backLink" class="govuk-back-link">
+    Back
+</a>
+
+<main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds govuk-body">
+            <partial name="_ErrorSummary" model="@ModelState" />
+            <form method="post">
+                <div class="govuk-form-group @(devolvedHasError ? "govuk-form-group--error" : "")">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-fieldset__heading">
+                                @ViewData["Title"]
+                            </h1>
+                        </legend>
+                        @if (devolvedHasError)
+                        {
+                            <p id="devolved-error" class="govuk-error-message">
```

Comment:
Need an aria-describedby on the fieldset matching `devolved-error`, when there is on

Reply:
1) Added aria-describedby to the <fieldset> to reference the hint (devolved-hint) and error message (devolved-error) conditionally.
2) Each radio button now includes aria-describedby referencing both the hint (devolved-hint) and the error (devolved-error) dynamically.

Reply:
Applied to all the new pages.